README.md

```diff
@@ -1,5 +1,8 @@
 # GIR
 
+"[GObject introspection (abbreviated ‘GIR’)](https://developer.gnome.org/programming-guidelines/stable/introspection.html.en) is a system which extracts APIs from C code and produces binary type libraries which can be used by non-C language bindings, and other tools, to introspect or wrap the original C libraries. It uses a system of annotations in documentation comments in the C code to expose extra information about the APIs which is not machine readable from the code itself."
```

Comment:
Why in quotes?

```suggestion
"[GObject introspection (abbreviated ‘GIR’)](https://developer.gnome.org/programming-guidelines/stable/introspection.html.en) is a system which extracts APIs from C code and produces binary type libraries and XML API descriptions which can be used by non-C language bindings, and other tools, to introspect or wrap the original C libraries. It uses a system of annotations in documentation comments in the C code to expose extra information about the APIs which is not machine readable from the code itself."
```

Reply:
The double quotes, are because this is directly quoted from the URL I provided.  The single quotes around GIR aren't needed, just my personal preference.  Thank you for the additional text.  I suppose you might want to include it as '[and XML API descriptions]' (i.e. in square brackets) to show that it's not part of the original quote.  OR if you want, just remove the outer double quotes, as it's no longer a direct quote.  I don't know the right thing to do here.  ....Just that I spend a whole bunch of time trying to figure out what the heck GIR was, so I could even start to read this.  .... And while you're at it please clarify the term 'crate', thought I now see it's from the rust packager or something.  Thanks again!